language/move-lang/src/expansion/translate.rs

```diff
@@ -48,19 +51,6 @@ impl Context {
         }
     }
 
-    fn error(&mut self, e: Vec<(Loc, impl Into<String>)>) {
```

Comment:
I probably like just dispatching out to `context.env.error` a lot better than my initial idea of doing some trait nonsense. Glad to see this